static/css/drawer.css

```diff
@@ -68,18 +68,11 @@ input[type=number]:invalid {
   box-sizing: border-box;
 }
 
-#treatmentDrawer .left-column {
-  display: block;
-  width: 100%;
-}
-
-#treatmentDrawer .left-column span {
-  width: 110px;
-  display: inline-block;
-}
-
-#treatmentDrawer .left-column input, #treatmentDrawer .left-column select {
-  width: 140px;
+#treatmentDrawer .w260 {
```

Comment:
the name of the class should be semantic not the name of the width, something like `wide` would be better